README.md

```diff
@@ -2,3 +2,94 @@
 Templates for use in the Domain Connect Protocol
 
 These map to the individual service providers for domain connect. See https://domainconnect.org
+
+# Example Template Format
+
+For details on how to constuct a Domain Connect template, refer to section 5.2 and 5.3 fo the Domain Connect Spec:
+
+https://github.com/Domain-Connect/spec/blob/master/Domain%20Connect%20Spec%20Draft.adoc#template-definition
+https://github.com/Domain-Connect/spec/blob/master/Domain%20Connect%20Spec%20Draft.adoc#template-record
+
+Templates should be named according the following pattern: <providerId>.<serviceId>.json
+For example: myprovider.com.website.json
+
+Following is an example of a complete Domain Connect template, with examples of various DNS records included:
+
+{
```

Comment:
Would be great adding some markdown to indicate code section, otherwise JSON is not readible.